builder/proxmox/config.go

```diff
@@ -87,6 +90,15 @@ type vgaConfig struct {
 	Type   string `mapstructure:"type"`
 	Memory int    `mapstructure:"memory"`
 }
+type storageConfig struct {
+	common.ISOConfig `mapstructure:",squash"`
+	DeviceType       string `mapstructure:"device_type"`
```

Comment:
I think I had a couple of comments around this last time round, which addressed the naming here in different ways and we got a bit of a mix of all now :smile: I think it's great that we no longer have the number as a separate field, but now it should probably be called `device` rather than `device_type`.

Reply:
That makes sense. Will change it shortly.